src/sections/Meshmap/meshmap-modes.js

```diff
@@ -92,9 +92,9 @@ const MeshmapModesWrapper = styled.div`
 
       .mode-name{
           top: 15%;
-          transition: all 1.5s ease 0.75s;
+          transition: all 0.75s ease 0.75s;
```

Comment:
This is the actual change. Ignore other changes.

Reply:
Please refrain from such "unwanted" changes next time. It adds unnecessary clutter to the PR, and reviewer's effort.